WooCommerce/Classes/ViewRelated/Products/Edit Product/EditCategories/ProductCategoryViewModelBuilder.swift

```diff
@@ -0,0 +1,103 @@
+import Foundation
+import Yosemite
+
+/// Creates `ProductCategoryViewModel` types
+///
+struct ProductCategoryViewModelBuilder {
```

Comment:
Hmm, since these don't contain any sense of identity I normally use `structs`. I guess they could be `enums` too 🤔 

But if the convention of the project is to use structs only for "data objects" I can change it!

Reply:
I'm inclined to use `enum` but it just feels wrong. Let's leave it as is.